Services/GraphApiOperationService.cs

```diff
@@ -23,12 +23,12 @@ public GraphApiOperationService(HttpClient httpClient, IOptions<WebOptions> webO
             webOptions = webOptionValue.Value;
         }
 
-        public async Task<dynamic> CallOnBehalfOfUserAsync(string accessToken)
+        public async Task<dynamic> GetUserInformation(string accessToken)
         {
             httpClient.DefaultRequestHeaders.Authorization =
-                new AuthenticationHeaderValue(Constants.AuthorizationScheme,
+                new AuthenticationHeaderValue(Constants.BearerAuthorizationScheme,
                                               accessToken);
-            var response = await httpClient.GetAsync(webOptions.GraphApiUrl);
+            var response = await httpClient.GetAsync($"{webOptions.GraphApiUrl}/beta/me");
```

Comment:
Proposing to have only the Microsoft Graph base URL in the config file
(this way the sample can also work in national and sovereign clouds)